src/OrchardCore.Modules/OrchardCore.Media.AmazonS3/Startup.cs

```diff
@@ -130,8 +129,8 @@ public override void ConfigureServices(IServiceCollection services)
         }
     }
 
-    private string GetMediaCachePath(IWebHostEnvironment hostingEnvironment,
-        string assetsPath, ShellSettings shellSettings)
-        => PathExtensions.Combine(hostingEnvironment.WebRootPath,
-            assetsPath, shellSettings.Name);
+    private static string GetMediaCachePath(IWebHostEnvironment hostingEnvironment, ShellSettings shellSettings, string assetsPath)
+    {
+        return PathExtensions.Combine(hostingEnvironment.WebRootPath, shellSettings.Name, assetsPath);
```

Comment:
Why you removed the inline body ;)

Reply:
I don't remember, maybe when I did the same change (the position of the settings name in the path) for media azure, and copy pasted from it ;)

Reply:
Done in both places ;)